src/SamplesApp/LinkedFiles/WebContent/js/site.js

```diff
@@ -0,0 +1,22 @@
+function sendWebMessage(backgroundColor) {
```

Comment:
Should be make this part of the documentation?

Reply:
It is there, in the previous section of the `.md` 👍